src/baseTrie.js

```diff
@@ -24,752 +22,759 @@ module.exports = Trie
   }
-  return node.raw
-}
 
-/**
- * The `data` event is given an `Object` hat has two properties; the `key` and the `value`. Both should be Buffers.
- * @method createReadStream
- * @memberof Trie
- * @return {stream.Readable} Returns a [stream](https://nodejs.org/dist/latest-v5.x/docs/api/stream.html#stream_class_stream_readable) of the contents of the `trie`
- */
-Trie.prototype.createReadStream = function () {
-  return new ReadStream(this)
-}
-
-// creates a new trie backed by the same db
-// and starting at the same root
-Trie.prototype.copy = function () {
-  return new Trie(this.db, this.root)
-}
+  // creates a new trie backed by the same db
+  // and starting at the same root
+  copy () {
+    return new Trie(this.db, this.root)
+  }
 
-/**
- * The given hash of operations (key additions or deletions) are executed on the DB
- * @method batch
- * @memberof Trie
- * @example
- * var ops = [
- *    { type: 'del', key: 'father' }
- *  , { type: 'put', key: 'name', value: 'Yuri Irsenovich Kim' }
- *  , { type: 'put', key: 'dob', value: '16 February 1941' }
- *  , { type: 'put', key: 'spouse', value: 'Kim Young-sook' }
- *  , { type: 'put', key: 'occupation', value: 'Clown' }
- * ]
- * trie.batch(ops)
- * @param {Array} ops
- * @param {Function} cb
- */
-Trie.prototype.batch = function (ops, cb) {
-  var self = this
-
-  async.eachSeries(ops, function (op, cb2) {
-    if (op.type === 'put') {
-      self.put(op.key, op.value, cb2)
-    } else if (op.type === 'del') {
-      self.del(op.key, cb2)
-    } else {
-      cb2()
-    }
-  }, cb)
-}
+  /**
+   * The given hash of operations (key additions or deletions) are executed on the DB
+   * @method batch
+   * @memberof Trie
+   * @example
+   * var ops = [
+   *    { type: 'del', key: 'father' }
+   *  , { type: 'put', key: 'name', value: 'Yuri Irsenovich Kim' }
+   *  , { type: 'put', key: 'dob', value: '16 February 1941' }
+   *  , { type: 'put', key: 'spouse', value: 'Kim Young-sook' }
+   *  , { type: 'put', key: 'occupation', value: 'Clown' }
+   * ]
+   * trie.batch(ops)
+   * @param {Array} ops
+   * @param {Function} cb
+   */
+  batch (ops, cb) {
+    async.eachSeries(ops, (op, cb2) => {
+      if (op.type === 'put') {
+        this.put(op.key, op.value, cb2)
+      } else if (op.type === 'del') {
+        this.del(op.key, cb2)
+      } else {
+        cb2()
+      }
+    }, cb)
+  }
 
-/**
- * Checks if a given root exists
- * @method checkRoot
- * @memberof Trie
- * @param {Buffer} root
- * @param {Function} cb
- */
-Trie.prototype.checkRoot = function (root, cb) {
-  root = ethUtil.toBuffer(root)
-  this._lookupNode(root, function (value) {
-    cb(null, !!value)
-  })
+  /**
+   * Checks if a given root exists
+   * @method checkRoot
+   * @memberof Trie
+   * @param {Buffer} root
+   * @param {Function} cb
+   */
+  checkRoot (root, cb) {
+    root = ethUtil.toBuffer(root)
+    this._lookupNode(root, value => {
+      cb(null, !!value)
+    })
+  }
 }
```

Comment:
Remaining functions equivalent as well, done by side-by-side comparison scrolling down editor on left and changes here on the right.